src/commands/commerce/extension/map.ts

```diff
@@ -147,4 +147,33 @@ export class MapExtension extends SfdxCommand {
             return returnResult;
         }
     }
+    private deleteduplicateMaps(extensionName: string, userName: string): void {
+        try {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+            const EPNQuery = forceDataSoql(
+                `SELECT ExtensionPointName FROM RegisteredExternalService WHERE DeveloperName='${extensionName}'`,
+                userName
+            ).result.records[0]['ExtensionPointName'];
+            const existingIds = forceDataSoql(
+                // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+                `SELECT DeveloperName,ExternalServiceProviderType FROM RegisteredExternalService WHERE ExtensionPointName='${EPNQuery}' AND ExternalServiceProviderType='Extension'`,
+                userName
+            );
+            for (const element of existingIds.result.records) {
+                const id = (element['ExternalServiceProviderType'] as string)
+                    .concat('__' as string)
+                    .concat(element['DeveloperName'] as string);
+                try {
+                    const deleteId = forceDataSoql(
+                        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+                        `SELECT Id FROM StoreIntegratedService WHERE Integration='${id}'`,
+                        userName
+                    ).result.records[0].Id;
+                    forceDataRecordDelete('StoreIntegratedService', deleteId, this.org.getUsername());
+                    // eslint-disable-next-line no-empty
```

Comment:
Would we consider removing the json log when deleting a record which looks like this
```
{
  "status": 0,
  "result": {
    "id": "1ffxx00000007CnAAI",
    "success": true,
    "errors": []
  },
  "warnings": [
    "apiVersion configuration overridden at \"55.0\""
  ]
}
``` 

and replacing it with a more descriptive message like "Unmapped the previous mapped extension 'element['DeveloperName']'  for this EPN" so users are notified that they're previous mapped extension has been unmapped